ledger/distribution/Cargo.toml

```diff
@@ -22,8 +22,8 @@ displaydoc = "0.2"
 protobuf = "2.27.1"
 retry = "2.0"
 # TODO: Replace with https://github.com/awslabs/aws-sdk-rust when it is ready.
-rusoto_core = { version = "0.48.0", features = ["rustls"], default_features = false }
-rusoto_s3 = { version = "0.48.0", features = ["rustls"], default_features = false }
+rusoto_core = { git = "https://github.com/legokichi/rusoto.git", branch = "enable-http1", features = ["rustls"], default_features = false }
```

Comment:
That's a pretty old branch off somebodies personal fork, I'm not sure using this is a good from a sec and update practice.  Maybe if we fork the upstream and make adjustments?

I'm looking at https://rusoto.github.io/rusoto/rusoto_core/region/enum.Region.html

There seems to be an option for custom region and their example uses a http endpoint, does this work?

Reply:
That's the same version as 0.48.0

Reply:
it's only 1 commit ahead on branch "enable-http1"

Reply:
Happy to perform the fork on our end and use that instead if that's preferable. Their example does not work, as they made changes in 0.48.0 that disabled the ability to use http connections. I believe it was unintentional but the dev that's in charge of rusoto has been unresponsive.

Reply:
Oh well the upstream hasn't been updated in a while either :)

Reply:
Well that's a shame on the broken upstream.   I'll leave the using this fork vs making our own to @jcape.

Reply:
Already forked it and updated, so it should be good :)